mungegithub/github/github.go

```diff
@@ -143,8 +143,8 @@ type Config struct {
 	Org      string
 	Project  string
 
-	state  string
-	labels []string
+	State  string
+	Labels []string
 
 	Token     string
```

Comment:
Would be hard and may not matter much at the moment, so feel free to
ignore. Maybe add a comment.

On Thu, Aug 11, 2016 at 3:49 PM, Daniel Smith <notifications@github.com>
wrote:

> In mungegithub/github/github.go
> <https://github.com/kubernetes/contrib/pull/1495#discussion_r74518008>:
>
> > @@ -143,8 +143,8 @@ type Config struct {
> >  	Org      string
> >  	Project  string
> >
> > -	state  string
> > -	labels []string
> > +	State  string
> > +	Labels []string
> >
> >  	Token     string
>
> probably should redact the contents of Token...
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/contrib/pull/1495/files/8714e01db9d88df7c880f2eb915dae2e818ed545#r74518008>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAnglvtucm9rRUINU3Lx-GA9003FzVDyks5qe6b4gaJpZM4Jebml>
> .
>